packages/ext-angular-modern-kitchensink/src/app/view/landingpage/landingpage.component.ts

```diff
@@ -1,13 +1,26 @@
 import { Component, OnInit, ChangeDetectorRef, ViewChild, ViewContainerRef, ComponentFactoryResolver, ComponentRef, ComponentFactory } from '@angular/core';
-import {navTreeRoot} from '../../../examples/index';
-
+import { navTreeRoot } from '../../../examples/index';
 import { Location } from '@angular/common';
 import { Router, NavigationEnd } from '@angular/router';
-
-import {ButtonComponent} from '../../../examples/Button/Button';
+import { ButtonComponent } from '../../../examples/Button/Button';
 import { VERSION } from '@angular/core';
+import { Subject } from "rxjs";
 
 declare var Ext: any;
+declare var _code: any;
+
+const generateBreadcrumb = (node) => {
+  const path = [];
+  do {
+    path.unshift({
+      isLeaf: !node.parentNode,
```

Comment:
This also does not seem to be correct. If node does not have children then it is a leaf.